docs/src/main/asciidoc/doc-reference.adoc

```diff
@@ -234,6 +243,10 @@ https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[AsciiDoc Sy
 The above source produces this link:https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/[AsciiDoc Syntax Quick Reference,window=_blank,opts=nofollow].
 =====
 
+[[cross-references]]
+=== Cross-references
+
+
 [[cross-references]]
 === Cross-references
```

Comment:
Per Erin's comment above "Cross-references" is repeated. Perhaps this second instance can be removed.

Reply:
Thanks, @Sheila. Oops. It is now removed :+1: